File: scripts/check-neon.js
const { Pool } = require("pg");
const p = new Pool({
  connectionString: "postgresql://neondb_owner:npg_SPahbB68xqur@ep-cold-hat-alaqlcr5.c-3.eu-central-1.aws.neon.tech/neondb?sslmode=require",
  ssl: { rejectUnauthorized: false },
});

async function check() {
  try {
    const tables = await p.query("SELECT tablename FROM pg_tables WHERE schemaname='public'");
    console.log("Tables:", tables.rows.map(r => r.tablename).join(", ") || "NONE");

    try {
      const count = await p.query("SELECT COUNT(*) as c FROM products");
      console.log("Products:", count.rows[0].c);
    } catch (e) {
      console.log("Products table: not yet created");
    }

    try {
      const cats = await p.query("SELECT COUNT(*) as c FROM categories");
      console.log("Categories:", cats.rows[0].c);
    } catch (e) {}

    try {
      const ship = await p.query("SELECT COUNT(*) as c FROM shipping_rates");
      console.log("Shipping rates:", ship.rows[0].c);
    } catch (e) {}
  } catch (e) {
    console.log("Connection error:", e.message);
  }
  p.end();
}
check();

File: check-neon.py
import psycopg2

NEON_URL = 'postgresql://neondb_owner:npg_SPahbB68xqur@ep-cold-hat-alaqlcr5.c-3.eu-central-1.aws.neon.tech/neondb?sslmode=require'

conn = psycopg2.connect(NEON_URL)
cur = conn.cursor()

cur.execute('SELECT COUNT(*) FROM products')
total = cur.fetchone()[0]

cur.execute("SELECT column_name FROM information_schema.columns WHERE table_name='products' AND column_name='title_ro'")
has_ro = cur.fetchone()

ro_count = 0
if has_ro:
    cur.execute("SELECT COUNT(*) FROM products WHERE title_ro IS NOT NULL AND title_ro != ''")
    ro_count = cur.fetchone()[0]

print(f'NeonDB: {total:,} produse | title_ro existe: {bool(has_ro)} | completate: {ro_count:,}')

cur.execute('SELECT title, title_ro FROM products LIMIT 3')
for t, tro in cur.fetchall():
    print(f'  EN: {(t or "")[:50]}')
    print(f'  RO: {tro}')
    print()

conn.close()
